### DISCUSSION TITLE
Release v0.30.0

### DISCUSSION THREAD

**Discussion by mrmonday** [libpnet/libpnet#565] [Announcements]
## What's Changed
* pnet_macros: fix num_bytes calculation by @yu-re-ka in https://github.com/libpnet/libpnet/pull/547
* Assorted cleanup by @mrmonday in https://github.com/libpnet/libpnet/pull/548
* support Vec with construct_with by @yu-re-ka in https://github.com/libpnet/libpnet/pull/550
* Misc fixes by @mrmonday in https://github.com/libpnet/libpnet/pull/551
* Update ipnetwork requirement from 0.18.0 to 0.19.0 by @dependabot in https://github.com/libpnet/libpnet/pull/552
* Fix build on architectures with unsigned char by @pkubaj in https://github.com/libpnet/libpnet/pull/561
* Changes to Functions to include inline and moved benches to criterion to use stable toolchain by @infosechoudini in https://github.com/libpnet/libpnet/pull/555
* Add illumos support by @teutat3s in https://github.com/libpnet/libpnet/pull/549
* enable #[no_std] use by providing std feature by @Felix-El in https://github.com/libpnet/libpnet/pull/562

## New Contributors
* @pkubaj made their first contribution in https://github.com/libpnet/libpnet/pull/561
* @infosechoudini made their first contribution in https://github.com/libpnet/libpnet/pull/555
* @teutat3s made their first contribution in https://github.com/libpnet/libpnet/pull/549
* @Felix-El made their first contribution in https://github.com/libpnet/libpnet/pull/562

**Full Changelog**: https://github.com/libpnet/libpnet/compare/v0.29.0...v0.30.0

<hr /><em>This discussion was created from the release <a href='https://github.com/libpnet/libpnet/releases/tag/v0.30.0'>Release v0.30.0</a>.</em>

**Comment by stappersg** [comment]
> This discussion was created from the release [Release v0.30.0](https://github.com/libpnet/libpnet/releases/tag/v0.30.0).

And further feeding the beast named _github dot com_:
```text
17:10 < mrmonday> v0.30.0 published
17:12 < mrmonday> neat, github can auto-generate release notes
17:12 < mrmonday> wish I'd known that before I made the release commit, could have stuck it in the commit message
17:12 < mrmonday> ah well
17:21  * stappers felt the need to quote Eric S. Raymond "Release Early, Release Often"
```

The idea is having a discussion in this _github dot com created discussion_,
way more easy then to prevent _auto created discussions_.
